content/en/docs/ambient/usage/debug-waypoint/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Debug connectivity issues with waypoints
+description: How to investigate problems routing through waypoint proxies.
+weight: 70
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+This guide describes what to do if you have enrolled a workload in a waypoint proxy, but you are not seeing the expected behavior.
+
+## Problems with authentication policy
```

Comment:
The guide is not only written for authn policy.  maybe reword it to L7 policy not enforced/processed by waypoint?

Reply:
The underlying problem is that Istio only ever used the word "policy" for security, but Gateway API uses it for "traffic routing" also.